config/app.php

```diff
@@ -37,7 +39,7 @@
     |
     */
 
-    'debug' => env('APP_DEBUG', false),
+    'debug' => (bool) env('APP_DEBUG', false),
```

Comment:
Oooo smart!

Reply:
It's like that in updated Framework, so not my official doing, but I do agree it is smart!

Reply:
It was all you, Diego!